### ISSUE TITLE
Add more prominent notification that I've been tipped

### ISSUE THREAD

**Issue by PixelantDesign** [gitcoinco/web#546]
As a user, I'd like to be notified via the app that i've been tip

Claim your tips with total amount...

**Comment by owocki** [comment]
just an FYI -- when a tip is received one will receive an email letting them know how to redeem their tip

**Comment by PixelantDesign** [comment]
@owocki, Yep- I was thinking notification could be more prominent in app.
